webapp/accounts/urls.py

```diff
@@ -0,0 +1,23 @@
+from django.conf.urls import url
+from .import views
+from django.contrib.auth.views import (
+    login, logout,
+    password_reset, password_change_done,
+    password_reset_confirm
+)
+
+
+urlpatterns = [
+    url(r'^$', views.home),
+    url(r'^login/$', login , {'template_name': 'accounts/login.html'}, name='login'),
```

Comment:
These regex's shouldn't need a trailing `/`.
URL's in terms of RESTful resources shouldn't end with trailing `/`s.
Please remove them from resources that don't need them.